src/main/resources/changesets/changelog-consent-86.0.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="86.0" author="connorlbark">
+        <!--Add mapping and dataset property type columns -->
+        <addColumn tableName="datasetproperty">
+            <column name="propertytype" type="varchar(255)"/>
+            <column name="schemaproperty" type="varchar(255)"/>
+        </addColumn>
```

Comment:
Let's snake case these like we're doing for all new postgres work:
```suggestion
        <addColumn tableName="dataset_property">
            <column name="property_type" type="varchar(255)"/>
            <column name="schema_property" type="varchar(255)"/>
        </addColumn>
```

Reply:
I think I commented too greedily ... `<addColumn tableName="datasetproperty">` is correct since that's the actual table name, not `dataset_property`